packages/babel-helpers/src/helpers-generated.ts

```diff
@@ -348,20 +348,20 @@ const helpers: Record<string, Helper> = {
   ),
-  // size: 593, gzip size: 303
+  // size: 593, gzip size: 305
   set: helper(
     "7.0.0-beta.0",
-    'import superPropBase from"superPropBase";import defineProperty from"defineProperty";function set(e,r,t,o){return set="undefined"!=typeof Reflect&&Reflect.set?Reflect.set:function(e,r,t,o){var f,i=superPropBase(e,r);if(i){if((f=Object.getOwnPropertyDescriptor(i,r)).set)return f.set.call(o,t),!0;if(!f.writable)return!1}if(f=Object.getOwnPropertyDescriptor(o,r)){if(!f.writable)return!1;f.value=t,Object.defineProperty(o,r,f)}else defineProperty(o,r,t);return!0},set(e,r,t,o)}export default function _set(e,r,t,o,f){if(!set(e,r,t,o||e)&&f)throw new TypeError("failed to set property");return t}',
+    'import defineProperty from"defineProperty";import superPropBase from"superPropBase";function set(e,r,t,o){return set="undefined"!=typeof Reflect&&Reflect.set?Reflect.set:function(e,r,t,o){var f,i=superPropBase(e,r);if(i){if((f=Object.getOwnPropertyDescriptor(i,r)).set)return f.set.call(o,t),!0;if(!f.writable)return!1}if(f=Object.getOwnPropertyDescriptor(o,r)){if(!f.writable)return!1;f.value=t,Object.defineProperty(o,r,f)}else defineProperty(o,r,t);return!0},set(e,r,t,o)}export default function _set(e,r,t,o,f){if(!set(e,r,t,o||e)&&f)throw new TypeError("failed to set property");return t}',
   ),
   // size: 193, gzip size: 179
   setFunctionName: helper(
     "7.23.6",
     'export default function setFunctionName(e,t,n){"symbol"==typeof t&&(t=(t=t.description)?"["+t+"]":"");try{Object.defineProperty(e,"name",{configurable:!0,value:n?n+" "+t:t})}catch(e){}return e}',
   ),
-  // size: 178, gzip size: 116
+  // size: 183, gzip size: 121
   setPrototypeOf: helper(
     "7.0.0-beta.0",
-    "export default function _setPrototypeOf(t,e){return _setPrototypeOf=Object.setPrototypeOf?Object.setPrototypeOf.bind():function(t,e){return t.__proto__=e,t},_setPrototypeOf(t,e)}",
+    "export default function _setPrototypeOf(t,e){return _setPrototypeOf=Object.setPrototypeOf?Object.setPrototypeOf.bind():function(t,e){return e&&(t.__proto__=e),t},_setPrototypeOf(t,e)}",
```

Comment:
At this point should `e` be guaranted to be defined as well? If so I can remove the if

Reply:
Yes!